Gemfile.lock

```diff
@@ -568,21 +568,20 @@ GEM
       sprockets (>= 2.8, < 4.0)
       sprockets-rails (>= 2.0, < 4.0)
       tilt (>= 1.1, < 3)
-    sassc (2.0.1)
+    sassc (2.2.0)
       ffi (~> 1.9)
-      rake
     scrub_rb (1.0.1)
     selenium-webdriver (3.142.3)
       childprocess (>= 0.5, < 2.0)
       rubyzip (~> 1.2, >= 1.2.2)
-    sidekiq (5.2.7)
-      connection_pool (~> 2.2, >= 2.2.2)
-      rack (>= 1.5.0)
-      rack-protection (>= 1.5.0)
-      redis (>= 3.3.5, < 5)
-    sidekiq-pro (4.0.5)
+    sidekiq (6.0.0)
```

Comment:
We may want to be weary about this also. https://github.com/mperham/sidekiq/blob/master/Changes.md#60

Reply:
Agreed. Let's work on that in a separate PR next week.